config.w32

```diff
@@ -232,61 +232,64 @@ if (PHP_MONGODB != "no") {
-
-    // Generated with: find src/libmongocrypt/src/crypto -name '*.c' -print0 | cut -sz -d / -f 5- | sort -dz | tr '\000' ' '
-    var PHP_MONGODB_MONGOCRYPT_CRYPTO_SOURCES="cng.c commoncrypto.c libcrypto.c none.c";
-
-    // Generated with: find src/libmongocrypt/src/os_posix -name '*.c' -print0 | cut -sz -d / -f 5- | sort -dz | tr '\000' ' '
-    var PHP_MONGODB_MONGOCRYPT_OS_POSIX_SOURCES="os_mutex.c os_once.c";
-
-    // Generated with: find src/libmongocrypt/src/os_win -name '*.c' -print0 | cut -sz -d / -f 5- | sort -dz | tr '\000' ' '
-    var PHP_MONGODB_MONGOCRYPT_OS_WIN_SOURCES="os_mutex.c os_once.c";
-
-    // Generated with: find src/libmongocrypt/kms-message/src -maxdepth 1 -name '*.c' -print0 | cut -sz -d / -f 5- | sort -dz | tr '\000' ' '
-    var PHP_MONGODB_MONGOCRYPT_KMS_MESSAGE_SOURCES="hexlify.c kms_b64.c kms_caller_identity_request.c kms_crypto_apple.c kms_crypto_libcrypto.c kms_crypto_none.c kms_crypto_windows.c kms_decrypt_request.c kms_encrypt_request.c kms_kv_list.c kms_message.c kms_request.c kms_request_opt.c kms_request_str.c kms_response.c kms_response_parser.c sort.c";
-
-    ADD_SOURCES(configure_module_dirname + "/src/libmongocrypt/src", PHP_MONGODB_MONGOCRYPT_SOURCES, "mongodb");
-    ADD_SOURCES(configure_module_dirname + "/src/libmongocrypt/src/crypto", PHP_MONGODB_MONGOCRYPT_CRYPTO_SOURCES, "mongodb");
-    ADD_SOURCES(configure_module_dirname + "/src/libmongocrypt/src/os_posix", PHP_MONGODB_MONGOCRYPT_OS_POSIX_SOURCES, "mongodb");
-    ADD_SOURCES(configure_module_dirname + "/src/libmongocrypt/src/os_win", PHP_MONGODB_MONGOCRYPT_OS_WIN_SOURCES, "mongodb");
-    ADD_SOURCES(configure_module_dirname + "/src/libmongocrypt/kms-message/src", PHP_MONGODB_MONGOCRYPT_KMS_MESSAGE_SOURCES, "mongodb");
-
-    var mongocrypt_version = mongodb_parse_version_file(configure_module_dirname + "/src/LIBMONGOCRYPT_VERSION_CURRENT", "")
-    mongocrypt_opts.MONGOCRYPT_BUILD_VERSION = mongocrypt_version.VERSION;
-
-    mongodb_generate_header(
-      configure_module_dirname + "/src/libmongocrypt/src/mongocrypt-config.h.in",
-      configure_module_dirname + "/src/libmongocrypt/src/mongocrypt-config.h",
-      mongocrypt_opts
-    );
-
-    mongodb_generate_header(
-      configure_module_dirname + "/src/libmongocrypt/src/mongocrypt.h.in",
-      configure_module_dirname + "/src/libmongocrypt/src/mongocrypt.h",
-      mongocrypt_opts
-    );
-  } else if (PHP_MONGODB_CLIENT_SIDE_ENCRYPTION != "no") {
-    WARNING("mongodb libmongocrypt support not enabled, crypto libs not found");
-
+  if (PHP_MONGODB_CLIENT_SIDE_ENCRYPTION != "no") {
+    if (mongoc_ssl_found) {
```

Comment:
I think this logic is correct, but I would have suggested the following to minimize the diff and conditional nesting:

```
if (PHP_MONGODB_CLIENT_SIDE_ENCRYPTION != "no" && mongoc_ssl_found) {
  // the big block of directives to enable libmongocrypt
} else if (PHP_MONGODB_CLIENT_SIDE_ENCRYPTION != "no") {
  // No SSL library found, we can't enable libmongocrypt
  WARNING("mongodb libmongocrypt support not enabled, crypto libs not found");
}

if (PHP_MONGODB_CLIENT_SIDE_ENCRYPTION == "no" && mongoc_ssl_found) {
  // Add kms_message sources bundled with libmongoc
  ADD_SOURCES(configure_module_dirname + "/src/libmongoc/src/kms_message/src", PHP_MONGODB_KMS_MESSAGE_SOURCES, "mongodb");
  ADD_FLAG("CFLAGS_MONGODB", "/I" + configure_module_dirname + "/src/libmongoc/src/kms-message/src");
}
```

Either works for me, and I appreciate the extra comment you added ("No SSL library found, we can't enable libmongocrypt") before the warning.